doc/data/messages/b/bad-super-call/bad.py

```diff
@@ -2,6 +2,11 @@ class Animal:
     pass
 
 
+class Fish:
```

Comment:
```suggestion
class Fish(Animal):
```
Shouldn't Fish inherit from Animal ?

Reply:
Yeah, but then this example doesn't work.

I'll change!

```suggestion
class Tree:
```

Reply:
I did suggest tree at first too but the Niece/parent example made me think this should work with fish/animal ?

Reply:
Don't you think this is clearer?

Reply:
Let's wait for the implementation of #6961 before adding that.